Demo/Application/Features/3D Secure/BraintreeDemoThreeDSecurePaymentFlowViewController.m

```diff
@@ -197,7 +197,7 @@ - (void)tappedToVerifyNewCard {
             self.callbackCount++;
             [self updateCallbackCount];
             if (error) {
-                if (error.code == BTPaymentFlowErrorTypeCanceled) {
+                if (error.code == 5) {
```

Comment:
Can we assign `5` to a named variable?

Reply:
Yeh, this change is annoying since merchants no longer have access to BT code constants.

Throughout the re-write we aren't making [these Error files](https://github.com/braintree/braintree_ios/blob/convert-payment-flow-client/Sources/BraintreePaymentFlow/BTPaymentFlowError.swift) per module public. If we did, merchants could easily access these error codes like `BTPaymentFlowError.canceled.code`. I think we should audit/revisit the error strategy throughout each module.

Another thing too, this format was kept to for compatibility with [`NSError`](https://developer.apple.com/documentation/foundation/nserror) which has code & domain. The only property [`Error`](https://developer.apple.com/documentation/swift/error) has is a localized description.

Reply:
How important do we think error codes are to merchants, or are detailed error strings sufficient?

Reply:
Addressed in the demo!

If we get merchant feedback re: wanting to expose the error codes, we can easily make them public.